go/stellar/stellarsvc/service_test.go

```diff
@@ -1223,6 +1225,7 @@ func TestMakeAccountMobileOnlyOnRecentMobile(t *testing.T) {
 	// this should not post a new bundle
 	err = tc.Srv.SetAccountMobileOnlyLocal(ctx, stellar1.SetAccountMobileOnlyLocalArg{
 		AccountID: a1,
+		Enabled:   true,
```

Comment:
there are other places in this file (just one or two) where we should be calling like this to turn off mobile only, but we're not because it didn't exist yet. currently looks likes this: 
```go
err = remote.MakeAccountAllDevices(ctx, g, a1)
```
but should maybe probably look like this:
```go
err = tc.Srv.SetAccountMobileOnlyLocal(ctx, stellar1.SetAccountMobileOnlyLocalArg{
    AccountID: a1,
    Enabled:   false,
}
```
could you update those too?